ynr/settings/base.py

```diff
@@ -412,7 +412,7 @@ def root(*path):
 
 RESULTS_FEATURE_ACTIVE = True
 CAN_EDIT_ELECTIONS = False
-FORMAT_MODULE_PATH = "settings.formats"
+FORMAT_MODULE_PATH = "ynr.formats"
```

Comment:
Is this right? That module doesn't exist, I don't think?

Reply:
I followed this format https://docs.djangoproject.com/en/4.1/topics/i18n/formatting/#creating-custom-format-files and, it's being found this way.

Reply:
In that example, the `formats` file is directly inside `mysite` (in the directory tree code block). We have `formats in `ynr/settings/constants/`, so the python path would be `ynr.settings.constants.formats`